packages/web/src/features/translation/TranslationView.tsx

```diff
@@ -327,6 +330,19 @@ export default function TranslationView() {
         onVerseChange={(verseId) =>
           navigate(`/interlinear/${language}/verses/${verseId}`)
         }
+        navigateToNextUnapprovedVerse={async () => {
+          const data = await apiClient.verses.findNextUnapprovedVerse(
+            verseId,
+            language
+          );
+          if (data && data.verseId) {
+            loadedFromNextButton.current = true;
```

Comment:
@arrocke This is so that when the translator navigates to the next unapproved verse, the first word of the verse is focused on. My reasoning was that this would make translation more convenient. Do you want me to remove this?

Reply:
Oh right, I forgot that this was how we did that. This is fine how you have it